### PR TITLE
Fix function comment to consistent with its name

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
Fix function comment to consistent with its name

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/network/dns/dns.go [hunk 1]
```diff
@@ -280,7 +280,7 @@ func getPodDNSType(pod *v1.Pod) (podDNSType, error) {
 	return podDNSCluster, fmt.Errorf(fmt.Sprintf("invalid DNSPolicy=%v", dnsPolicy))
 }
 
-// Merge DNS options. If duplicated, entries given by PodDNSConfigOption will
+// mergeDNSOptions merge DNS options. If duplicated, entries given by PodDNSConfigOption will
```
Comment (mkumatag) [inline pkg/kubelet/network/dns/dns.go]: s/merge/merges/

File: pkg/kubelet/pleg/generic.go [hunk 2]
```diff
@@ -337,7 +337,7 @@ func (g *GenericPLEG) cacheEnabled() bool {
 	return g.cache != nil
 }
 
-// Preserve an older cached status' pod IP if the new status has no pod IP
+// getPodIP preserve an older cached status' pod IP if the new status has no pod IP
```
Comment (mkumatag) [inline pkg/kubelet/pleg/generic.go]: s/preserve/preserves/

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: pkg/kubelet/network/dns/dns.go (hunk 1 above)
Comment (xichengliudui) [reply pkg/kubelet/network/dns/dns.go]: Thank you very much. I have modified it, is that right?

File: pkg/kubelet/pleg/generic.go (hunk 2 above)
Comment (xichengliudui) [reply pkg/kubelet/pleg/generic.go]: thanks!

File: pkg/kubelet/network/dns/dns.go (hunk 1 above)
Comment (mkumatag) [reply pkg/kubelet/network/dns/dns.go]: 👍

**Comment by xichengliudui** [comment]
@mkumatag  thanks.

**Comment by mkumatag** [comment]
> @mkumatag thanks.

@xichengliudui Now you need to chase the maintainers to get approved tag ;) all the best :)

**Review by neolit123 — COMMENTED** [review]
/kind cleanup
/priority backlog
/retest

**Pull request merged**